userapp/src/components/table/HelixTable.js

```diff
@@ -101,12 +64,35 @@ const HelixTable = ({
     setOrderBy(property)
   }
 
+  // searchFilter contains a func that store filter query search upon user input
+  const [searchFilter, setSearchFilter] = useState({ search: (users) => { return users }})
+  
+  /**
+   * @param {object} event the event object contains user input
+   * Pass the user query input to searchFilter and it store which object matches the query 
+   */
+  const onSearch = (event) => {
+    const { value } = event.target
+    setSearchFilter({ search: (users) => {
+        console.log(users)
+        if (value === '') return users
+        else
+          return users.filter((user) =>
+            (user.FirstName.toLowerCase().includes(value.toLowerCase()) ||
```

Comment:
pass in columns that can be sorted into table so that this is not hard coded in table, but configured where it is called